pkg/ovs/openflow/default.go

```diff
@@ -0,0 +1,31 @@
+// +build !windows
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package openflow
+
+import (
+	"os"
+	"path"
+)
+
+const defaultOVSRunDir = "/var/run/openvswitch"
+
+func getOVSBridgeAddress(brName string) string {
+	ovsRunDir := os.Getenv("OVS_RUN_DIR")
```

Comment:
Any reason to support reading from env?

Reply:
Wenying suggests that we need to consider the situation that users may use a different "OVS run path" other that our default value.

Reply:
But all other configurable parameters are from the config file. Any thing special for this one to read from env? @wenyingd

Reply:
@jianjuns @ruicao93 There is only one configuration item referring to OVSDB socket(named pipe), but no one for OpenFlow connection in current code. The OF connection is setup using the path here.

If not using an environment variable, it would be OK to add an parameter in Bridge.Connect, and share with the path configured for OVSDB socket(not the full path, but the directory path).

Besides, I don't think it is necessary to read from environment on Linux, as it is running in the container, and the path is relatively fixed. But for Windows, the path is changable. Another reason I suggest to read from enviroment, is because $env:OVS_RUNDIR is used by cloudbase when installing OVS with MSI. And if cloudbase is chosen as the default OVS on Windows, we could consume it directly.

Reply:
@wenyingd I think it's more better for user to manage the address in config file. Could we consider to add "OVS_RUNDDIR" param in config file and consume it both in ofclient and ovsdbclient?

Reply:
Yes, also feel is no good reason better to use config file, rather than introducing another way to populate config.